docs/rp2/quickref.rst

```diff
@@ -55,6 +55,60 @@ The :mod:`rp2` module::
 
     import rp2
 
+Networking
+----------
+
+WLAN
+^^^^
+
+.. note::
+   This section applies only to devices that include WiFi support, such as the `Pico W`_ and `Pico 2 W`_.
+
+The :class:`network.WLAN` class in the :mod:`network` module::
+
+    import network
+
+    wlan = network.WLAN(network.WLAN.IF_STA) # create station interface
```

Comment:
I suggest using just `wlan = network.WLAN()` because STA is (always) the default and it's simpler.

Reply:
OK, and will update the ESP32 quickref.

Reply:
Done.